### ISSUE TITLE
Install block at the SSOwat configuration

### ISSUE THREAD

**Issue by stilobique** [YunoHost-Apps/mopidy_ynh#8]
Hi,
i'm try to install this app on my personal server (a personal computer with the Yunohost 2.7.5 distrib install).
I can't install this apps without a the spotify user name (not optional field ?) When i'm install the package Yunohost stay blocked on the _The SSOwat configuration has been generated_.
I can attach the install log if you want.

I don't know all information needed to debug this issue are need. More information :

- Debian 8.10 64bit (Linux 3.16.0-4-amd64)
- Yunohost 2.7.5
- Install from the community package

Thanks for your jobs !

**Comment by siwinter** [comment]
The app has been completely redesigned.
If this does not solve the problem, please open a new issue.
